packages/server/src/interceptor.ts

```diff
@@ -0,0 +1,10 @@
+import type { Express } from 'express'
+import { App } from './app'
+
+export class Interceptor {
+  constructor(private app: App, private express: Express) {}
+
+  async setup() {
+    this.app.metrics.init(this.express)
+  }
+}
```

Comment:
Not really what I meant but whatever. I think it's time I unsubscribe from this repo lol I'll stop bothering you with nitpick comments